src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+public class Tagging {
```

Comment:
Missing header comment. All non-trivial classes/methods should have javadoc format header comments.